source/MaterialXGraphEditor/Main.cpp

```diff
@@ -112,7 +113,11 @@ int main(int argc, char* const argv[])
 
         if (token == "--material")
         {
-            materialFilename = nextToken;
+            if (nextToken != materialFilename)
+            {
+                materialFilename = nextToken;
```

Comment:
However if the user specifies the file on the command line, then this should cache the filename and allow CTRL-S and CTRL-R to work after load.